doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -59,6 +59,13 @@ Thread
 
   *  Support for the :ref:`thread_architectures_designs_cp_ncp` architecture and the related tools.
 
+Zigbee
+------
+
+* Added:
+
+  * CLI command for monitoring the list of active Zigbee neighbors.
```

Comment:
I think we should stick with `Zigbee Shell` instead of `CLI`
```suggestion
  * Zigbee Shell command for monitoring the list of active Zigbee neighbors.
```